WordPress/Classes/Models/Blog.m

```diff
@@ -400,6 +402,19 @@ - (NSSet *)allowedFileTypes
     return [NSSet setWithArray:allowedFileTypes];
 }
 
+- (void)setOptions:(NSDictionary *)options
+{
+    [self willChangeValueForKey:@"options"];
+    [self setPrimitiveValue:options forKey:@"options"];
+    self.jetpack = nil;
+    [self didChangeValueForKey:@"options"];
+}
+
++ (NSSet *)keyPathsForValuesAffectingJetpack
```

Comment:
It's for KVO, see https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/KeyValueObserving/Articles/KVODependentKeys.html#//apple_ref/doc/uid/20002179-BAJEAIEE